### ISSUE TITLE
HACS registration

### ISSUE THREAD

**Issue by S-Przybylski** [malcolmrigg/wizard-clock-card#8]
Dear @malcolmrigg,
my daugther and i love your clock! Currently i use it for displaying if someone is at home, at work or somewhere else.

I have a suggestion: Could you please register your beautiful clock at the HACS register (to be honest, i personally do not now how to proceed)? The benefit will be that your clock is more present to the HA community and also updates are quite easier to perform.

One remark to the ressource typ: Currently your plugin uses "java script file", which seems to be deprecated soon. Could you please switch to java script module instead (i am not a programmer - sorry if my question is missleading)?

Wishlist: Pictures inside the clock hands (at the leaves?), which could be exchanged by event / position optionally.

**Comment by malcolmrigg** [comment]
Hi, glad you like it! Thanks for the feedback 😄 

Adding it to HACS is something that's been suggested before, but as I haven't been using HACS I hadn't really been motivated to do that. However, I've recently installed HACS myself and so will be taking a look at this fairly soon if I get the time... Part of that work will probably be switching to a module and tidying up the config.

I've also got plans to update the component itself (I've brushed up my javascript skills a bit since I originally wrote this), and animated pictures (as in the original Weasley clock) is something I'd love to add. It may take me a few weeks depending on how busy I am...
